common/lib/xmodule/xmodule/tabs.py

```diff
@@ -121,14 +133,8 @@ def __setitem__(self, key, value):
 
         Note: the 'type' member can be 'get', but not 'set'.
```

Comment:
Did you determine changing the type is ok?  Is this the same type that is used to find the tab?